views/faucet.pug

```diff
@@ -77,7 +77,6 @@ block content
         window.location.hash = "#transaction";
       }
     }
-
     async function enableEthereum() {
```

Comment:
@hugo-dc added this back here, because not sure it should be removed. But if it does, I think it should be done in a separate PR.